magento/pwa-studio
# [PWA-1101] Add support for Configurable Product Image Setting

## Description

As a admin user, I would like the ability to change settings in the admin panel to control the configurable product image, and view it in my Venia storefront.

Admin Config Settings:

`Stores > Configuration > Sales > Checkout > Shopping Cart > Configurable Product Image`

## Related Issue





Closes https://jira.corp.magento.com/browse/PWA-1101

## Acceptance



### Verification Stakeholders
@dpatil-magento
@sirugh 

### Specification



### Verification Steps


1. Change admin configuration setting for `Stores > Configuration > Sales > Checkout > Shopping Cart > Configurable Product Image`
2. Save configuration and navigates to the Venia storefront
3. Add a Configurable Product to Cart and verify that product thumbnail corresponds admin configuration setting
4. Verify product thumbnail on Shopping Cart page
5. Verify product thumbnail on Checkout review page
6. Verify product thumbnail on Order Confirmation page
7. Verify product thumbnail on Order History page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
